lighthouse-core/report/html/renderer/util.js

```diff
@@ -213,7 +212,7 @@ class Util {
    */
   static formatMilliseconds(ms, granularity = 10) {
     const coarseTime = Math.round(ms / granularity) * granularity;
-    return `${coarseTime.toLocaleString(Util.numberDateLocale)}${NBSP}ms`;
+    return `${Util.numberFormatter.format(coarseTime)}${NBSP}ms`;
```

Comment:
Out of scope, but disappointing that we just append 'ms' or 's' here.  We need that unit's proposal to go through!

Part of the process of making this consistent:
![image](https://user-images.githubusercontent.com/6392995/63451814-d9365380-c3f9-11e9-9727-083fe0c5fe8c.png)